### DISCUSSION TITLE
Causal Inference Calculation Error

### DISCUSSION THREAD

**Discussion by Ad4102** [pgmpy/pgmpy#1468] [Q&A]
I am trying to perform a very simple causal query with my own model (a fork). However, it gives the following error.

ValueError: Variable: FN not found in arguments

FN is my outcome here.

**Comment by ankurankan** [comment]
@Ad4102 Thanks for reporting the issue. Would it also be possible to share your code so that I can reproduce the error?

**Reply by Ad4102** [reply]
@ankurankan thank a lot for swift response. 

I will share the code in a while. However the problem I have noticed is that it does not work for "fork network" formation. It works fine if a confounder is involved.
